tortoise_interpreter.py

```diff
@@ -1,11 +1,21 @@
-def main_tortoise(program): 
+def main_tortoise(program):
+    program = sanitize_program(program)
     COMMANDS = {'U': pen_up, 'D': pen_down, 'P': pen_colour, 'N': move_north, 'S': move_south, 'E': move_east, 'W': move_west}
     command_to_parse = navigate_command_functions(COMMANDS, program)
 
     for command in command_to_parse:
         print(command)
 
 
+def sanitize_program(program):
+    # If a command has two parts, the second element of the list will an empty string.
```

Comment:
Have you run this on all the example Tortoise programs that we have so far? Does this work on all of them? (spoiler alert: it won't :grimacing: )

Reply:
Oh I didn't realize it had to work on programs without comments... Oops